crates/cli/src/error.rs

```diff
@@ -230,9 +230,9 @@ impl<'a> fmt::Display for ErrorFormat<'a> {
     if let Some(url) = link {
       let reference = Style::new().bold().dimmed().paint("See also:");
       let link = format!(
```

Comment:
TODO @HerringtonDarkholme add a comment here to explain what's terminal link style.

Reply:
Yeah, I think we can do it in this PR. (instead of the replacement)